README.md

```diff
@@ -1,2 +1,249 @@
+should clearly identify any gaps that you are aware of.
+
+At this early stage, using git and GitHub is merely a suggestion. You may also
+choose other forms of exchanging ideas with your peers and collaborating on a
+text document.
+
+#### Opening a pull request
+
+Once you have built trust in your proposal and are ready to receive feedback
+from a wider audience, it is time to open a pull request on this repository.
+Once you have opened the PR, replace `0000` in the file name of your RFC with
+the PR number and insert a link to the PR at the top of your RFC.
+
+Opening a PR will allow the stakeholders and interested community members to
+review your proposal and raise concerns or make suggestions. We will also
+assign a [shepherd](#role-of-the-shepherd) to your PR who helps manage the
+remaining process. At this stage, the shepherd should request reviews from
+relevant stakeholders and community members that are affected by or generally
+interested in the RFC.
+
+#### Addressing feedback
+
+The reviewers will share their valuable thoughts and opinions and evaluate the
+RFC based on the [acceptance
+criteria]()https://github.com/lf-lang/.github/blob/main/CONTRIBUTING.md#acceptance-criteria.
+Aim to address this feedback and, if necessary, make changes to the RFC in order
+to help reach an agreement. The shepherd may help to moderate the discussion if
+needed. The acceptance criteria provide a guideline on
+
+Make changes as new commits to the pull request, and leave a comment on the pull
+request explaining your changes. Specifically, do not squash or rebase commits
+after they are visible on the pull request.
+
+Not every concern raised by a reviewer is an indication of a bad design and
+requires a significant change to the RFC. No solution is perfect, and usually,
+there are trade-offs involved. Thus, addressing a concern can be as simple as
+acknowledging it as a drawback in the RFC document. Again, timeliness and
+frequent iteration are more important than being perfect.
+
+If the RFC is discussed during a meeting, notes and a summary of the meeting
+should be shared as a comment in the PR thread.
+
+#### Final comments and decision
+
+After the discussion settles down, the shepherd sends out a final call for
+comments (FCP) with a disposition to accept or reject the RFC. The FCP lasts 7
+days and is announced both on the community mailing list and the [`#rfc` Zulip
+channel](https://lf-lang.zulipchat.com/#narrow/stream/425418-RFCs). The
+announcement should include the RFC title and abstract as well as a link to the
+PR. At the next developer meeting after the FCP period ends, the maintainers
+will discuss the RFC and reach a decision.
+
+#### Accepted RFCs
+
+If the RFC is accepted, the shepherd announces the new active RFC on the
+community mailing list and in the [`#rfc` Zulip channel](https://lf-lang.zulipchat.com/#narrow/stream/425418-RFCs).
+
+Furthermore, the shepherd creates an issue (or issues) that tracks the
```

Comment:
I would leave this as a recommendation or leave it out. This relies to much on discipline and is too costly if we try to enforce it.